src/EditorFeatures/TestUtilities/ServicesTestUtilities.csproj

```diff
@@ -94,6 +94,7 @@
     <InternalsVisibleToTest Include="Roslyn.Compilers.VisualBasic.Test.Utilities" />
     <InternalsVisibleToTest Include="Roslyn.InteractiveHost.UnitTests" />
     <InternalsVisibleToTest Include="Roslyn.Services.Editor.UnitTests" />
+	<InternalsVisibleToTest Include="Roslyn.VisualStudio.Test.Utilities" />
```

Comment:
Indention here is off; not sure if it's this line that's wrong or all the others.